package.json

```diff
@@ -27,7 +27,7 @@
   },
   "scripts": {
     "start": "npm run test:watch",
-    "test": "cross-env NODE_ENV=test NODE_PATH=./src babel-node node_modules/.bin/tape test/*.spec.js test/**/*.spec.js | tap-spec",
+    "test": "cross-env NODE_ENV=test NODE_PATH=./src babel-node test/index.js | tap-spec",
```

Comment:
There was some kind of reason, but I can't remember it at the moment.
I don't see a reason why not using the glob patterns here, I'll change that :)
Thanks for pointing this out 👍